bin/compile.js

```diff
@@ -15,6 +15,7 @@ program
     .option('-w, --watch', 'watch for changes and recompile')
     .option('-b, --bytecode', 'output bytecode')
     .option('-x, --no-babelify', 'skip Babel transforms')
+    .option('-d, --debug', 'enable debug in browserify')
```

Comment:
This is a breaking change, as the current default is to include it. How about `-S, --no-sourcemap` or something like that?